Content.Shared/Buckle/SharedBuckleSystem.Buckle.cs

```diff
@@ -357,7 +357,7 @@ public bool TryBuckle(EntityUid buckleUid, EntityUid userUid, EntityUid strapUid
         if (TryComp<AppearanceComponent>(buckleUid, out var appearance))
             Appearance.SetData(buckleUid, BuckleVisuals.Buckled, true, appearance);
 
-        _rotationVisuals.SetHorizontalAngle(buckleUid,  strapComp.Rotation);
+        _rotationVisuals.SetHorizontalAngle(buckleUid, Angle.FromDegrees(strapComp.Rotation));
 
```

Comment:
Good fix though you're better off making strapcomponent an angle to fix it permanently.